pkg/geo/geomfn/affine_transforms.go

```diff
@@ -281,3 +281,13 @@ func RotateWithXY(g geo.Geometry, rotRadians, x, y float64) (geo.Geometry, error
 		},
 	)
 }
+
+// TransScale returns a modified Geometry whose coordinates are
+// translate by delX and delY and scale by xFactor and yFactor
+func TransScale(g geo.Geometry, delX, delY, xFactor, yFactor float64) (geo.Geometry, error) {
+	g, err := Translate(g, []float64{delX, delY})
+	if err != nil {
+		return g, err
+	}
+	return Scale(g, []float64{xFactor, yFactor})
```

Comment:
we should actually speed to one operation by using the `affine` transformation:

```

return affine(
--
  | t,
  AffineMatrix([][]float64{
   {xFactor, 0, 0, xFactor * deltaX},
   {0, yFactor, 0, yFactor * deltaY},
   {0, 0, 1, 0},
   {0, 0, 0, 1},
   }),
   )


```

Reply:
I've tried to google it how to just use only using an affine matrix, but not luck